packages/amplify_flutter/lib/amplify.dart

```diff
@@ -37,13 +37,12 @@ export 'package:amplify_core/types/index.dart';
 part 'method_channel_amplify.dart';
 
 /// Top level singleton Amplify object.
-final AmplifyClass Amplify = new AmplifyClass._private();
+final AmplifyClass Amplify = new AmplifyClass();
 
 /// This is a private class and customers are not expected to
 /// instantiate an object of this class. Please use top level
 /// `Amplify` singleton object for making calls to methods of this class.
 class AmplifyClass extends PlatformInterface {
-  AmplifyClass._private();
```

Comment:
What is the reason for this change?

Reply:
That line causes a compile issue because PlatformInterface doesn't have an equivalent parameterless constructor.

Reply:
@Amplifiyer  what do you think of this?

Reply:
I'm not sure why it is causing the compile issue since we are not calling super's constructor. a named `_` constructor help us to keep this a singleton and only initialized by us. Making this constructor public will open a huge can of worms.

Reply:
This is the compile error we get if the AmplifyClass._private(); constructor is present: 

> The superclass 'PlatformInterface' doesn't have a zero argument constructor.

I did not add a public constructor.  In the existing code we have a constructor: 

`  AmplifyClass() : super(token: _token);
`

That was added on 1/5/21 in the PR refactor amplify_core to amplify_flutter

Reply:
I see that's just left over from a previous implementation (since the constructor is all tucked in the middle). You can still use the same thing with private constructor and pass an argument to super. 

```dart
class A {
  int superParameter;
  A(int p) {
    this.superParameter = p;
  }
  void printVal() {
    print(superParameter);
  }
}

class B extends A {
  B._private(): super(1);
}

var b = B._private();

void main() {
    b.printVal();
}
```

Reply:
Alright switched to private constructor

final AmplifyClass Amplify = new AmplifyClass._private();

AmplifyClass._private() : super(token: _token);